### PR TITLE
Path change

### PR DESCRIPTION
**Pull request by mickvanhulst**

### CODE REVIEW

File: REL/generate_train_test.py [hunk 1]
```diff
@@ -70,8 +70,7 @@ def process_wned(self, dataset):
         """
         split = "\n"
         annotations_xml = os.path.join(
-            self.wned_path, "{}/{}.xml".format(dataset, dataset)
-        )
+            self.wned_path, dataset, dataset, '.xml')
```
Comment (KDercksen) [inline REL/generate_train_test.py:73]: This will result in the equivalent of "dataset/.xml"

File: REL/generate_train_test.py [hunk 2]
```diff
@@ -15,8 +15,8 @@
 
 class GenTrainingTest(MentionDetectionBase):
     def __init__(self, base_url, wiki_version, wikipedia):
-        self.wned_path = "{}/generic/test_datasets/wned-datasets/".format(base_url)
-        self.aida_path = "{}/generic/test_datasets/AIDA/".format(base_url)
+        self.wned_path = os.path.join(base_url, '/generic/test_datasets/wned-datasets/')
+        self.aida_path = os.path.join(base_url, "{}/generic/test_datasets/AIDA/")
```
Comment (KDercksen) [inline REL/generate_train_test.py:19]: Can do without {}

File: REL/generate_train_test.py [hunk 3]
```diff
@@ -334,9 +331,7 @@ def __save(self, mentions_dataset, file_name):
         """
 
         with open(
-            "{}/{}/generated/test_train_data/{}.pkl".format(
-                self.base_url, self.wiki_version, file_name
-            ),
+            os.path.join(self.base_url, self.wiki_version, '/generated/test_train_data/', file_name, 'pkl'),
```
Comment (KDercksen) [inline REL/generate_train_test.py:334]: Same here, I don't think this results in "blabla.pkl". I think it should be `f"{file_name}.pkl"`

File: REL/training_datasets.py [hunk 4]
```diff
@@ -37,7 +37,7 @@ def load(self):
 
             print("Loading {}".format(ds))
             datasets[ds] = self.__read_pickle_file(
-                "{}/generated/test_train_data/{}.pkl".format(self.base_url, ds)
+                os.path.join(self.base_url, 'generated/test_train_data/', ds) + '.pkl'
```
Comment (KDercksen) [inline REL/training_datasets.py:40]: Inconsistency; `f"{ds}.pkl"` is probably neater

Comment (mickvanhulst) [reply REL/training_datasets.py:40]: Agreed!

**Pull request merged**